doc/HttpLuaModule.wiki

```diff
@@ -5581,6 +5581,73 @@ According to the current implementation, only the parent coroutine (or "light th
 
 This API was first enabled in the <code>v0.9.9</code> release.
 
+== ngx.semaphore.new ==
```

Comment:
We just need a simple reference here for the `ngx.semaphore` module in `lua-resty-core`.